pydm/widgets/frame.py

```diff
@@ -68,26 +68,7 @@ def alarm_severity_changed(self, new_alarm_severity):
         """
         if self._channel is None:
```

Comment:
Is the alarm_severity_changed method even needed at all on PyDMFrame anymore?

Reply:
Removed.